recipes/libsecret/all/conanfile.py

```diff
@@ -0,0 +1,86 @@
+from conans import ConanFile, Meson, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class LibsecretConan(ConanFile):
+    name = "libsecret"
+    description = "A library for storing and retrieving passwords and other secrets"
+    topics = ("conan", "libsecret", "gobject", "password", "secret")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://wiki.gnome.org/Projects/Libsecret"
+    license = "LGPL-2.1"
```

Comment:
```suggestion
    license = "LGPL-2.1-or-later"
```

Only LGPL-2.1 is considered deprecated.
https://spdx.org/licenses/LGPL-2.1-or-later.html